src/devices/Ht1632/Id.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Ht1632
+{
+    /// <summary>
+    /// Mode ID
+    /// </summary>
+    internal enum Id : byte
+    {
+        // References first 3-bits of command codes like:
+        // 100 0000-0000-X
+        // ~~~
+
+        /// <summary>
+        /// READ Mode
+        /// </summary>
+        READ = 0b_110,
```

Comment:
For internals, it's fine to keep read in uppercase if it's like this in the documentation.